### ISSUE TITLE
Is this code abandoned?

### ISSUE THREAD

**Issue by Brink67** [dtbaker/stylepress#35]
StylePress is an excellent idea. I like the way it is implemented much better than the 'Header Footer Elementor' plugin. I have not found anything else similar so far. If I have missed one please let me know.

 It's too bad that there have been no updates here for many months. And indeed the project has not moved beyond beta.

I would not mind paying for this this if it worked properly and had support. I understand that your statement was that StylePress would be free. So leave this beta version of StylePress free. Come up with another name and continue to develop it and charge a little money. You may be surprised at who would be willing to pay.

**Comment by HelmutGoebel** [comment]
I agree. The project is dead, because the author has never committed to it. Be patient a few mor eweeks and Elementor will incorporate themer functionality into Elementor Pro.

**Comment by Brink67** [comment]
Thanks for the reply. I have the Pro plugin for Elementor. I suppose we will just have to see what the code gods will grant us.

**Comment by HelmutGoebel** [comment]
That's how I would see it.

**Comment by aaronpolly1** [comment]
Honestly, if that's the case I'd rather move to divi then to a paid version of elementor pro.. but that's just my 0.02$
